azurerm/resource_arm_virtual_machine_scale_set_test.go

```diff
@@ -174,6 +174,67 @@ func TestUnitAzureRMVirtualMachineScaleSet_basicPublicIP_simpleUpdate(t *testing
 	})
 }
 
+func TestUnitAzureRMVirtualMachineScaleSet_updateNetworkProfile(t *testing.T) {
+	resourceName := "azurerm_virtual_machine_scale_set.test"
+	ri := tf.AccRandTimeInt()
+	location := testLocation()
+	config := testAccAzureRMVirtualMachineScaleSet_basicEmptyPublicIP(ri, location)
+	updatedConfig := testAccAzureRMVirtualMachineScaleSet_basicEmptyNetworkProfile_true_ipforwarding(ri, location)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMVirtualMachineScaleSetDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMVirtualMachineScaleSetExists(resourceName),
+					testCheckAzureRMVirtualMachineScaleSetIPForwarding(resourceName, false),
+				),
+			},
+			{
+				Config: updatedConfig,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMVirtualMachineScaleSetIPForwarding(resourceName, true),
+				),
+			},
+		},
+	})
+}
+
+func TestUnitAzureRMVirtualMachineScaleSet_updateNetworkProfile_ipconfiguration_dns_name_label(t *testing.T) {
```

Comment:
same thing here... can we update the test function name to start with `TestAccAzureRMVirtualMachineScaleSet` instead of `TestUnitAzureRMVirtualMachineScaleSet`